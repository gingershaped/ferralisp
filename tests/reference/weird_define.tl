(d minus-three (s 0 3))
(d -3.0 (q minus-three))
-3.0
(v -3.0)