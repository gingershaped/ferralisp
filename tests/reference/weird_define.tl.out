minus-three
-3.0
minus-three
-3